### PR TITLE
Template change to provide link to forum

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: layouts/user/content.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="td-content">
+	<h1>{{ .Title }}</h1>
+	{{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
+	{{ .Content }}
+  <hr />
+  Still have questions?  No problem!  Just head over to the
+<a href="https://www.zen-cart.com/forum.php">
+  main Zen Cart support forum</a> and ask there.
```
Comment (drbyte) [inline layouts/user/content.html]: Did you mean to code this text directly into the template? Or just call a shortcode from it?

File: layouts/shortcodes/faq_questions.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<! - Note this is in layouts/partials/faq_questions.html and layouts/shortcodes/faq_questions.html -->
```
Comment (drbyte) [inline layouts/shortcodes/faq_questions.html]: Why is this in both places?

Also, isn't the `<! -` supposed to be `2 hyphens` not `space-hyphen`?

Comment (scottcwilson) [reply layouts/shortcodes/faq_questions.html]: D-ohhh!I I thought I had fixed that.

File: layouts/user/content.html (hunk 1 above)
Comment (scottcwilson) [reply layouts/user/content.html]: You can't call a shortcode in a template.  That's why I had to duplicate the content of faq_questions.html.

**Comment by drbyte** [comment]
Is there still no way to reduce the duplication, by having one call the other?

**Comment by scottcwilson** [comment]
> Is there still no way to reduce the duplication, by having one call the other?

AFAIK no.

**Comment by scottcwilson** [comment]
Let's not get stuck on this though - it's a relatively small matter.  I can create an issue to be sure it gets tracked if you'd like.

**Comment by drbyte** [comment]
Okay, I think I understand now: the shortcode is used in all the `*_misc.md` files, and the partial is used in the template file when displaying a "list" (section table of contents) page.

**Pull request merged**